app/views/qualifications/certificates/_induction.html.erb

```diff
@@ -0,0 +1,19 @@
+<img src="/induction-certificate.jpg" class="header-image" alt="Department for Education" />
+
+<div class="content">
+  <h1 class="heading">Induction</h1>
+  <p class="text">This is to certify that: <strong><%= teacher.name %></strong></p>
+  <p class="text">Teacher Reference Number: <strong><%= teacher.trn %></strong></p>
+  <p>&#160;</p>
+  <p class="text">has successfully completed a statutory induction period at a maintained school,<br/>
+    maintained nursery school, maintained children’s centre, non-maintained special school,<br/>
+    independent school, academy school, 16-19 academy, sixth form college or further education<br/>
+    institution. The holder of this certificate is a qualified teacher and has obtained the required<br/>
+    qualifications and completed the necessary training for the profession of school teacher in England.</p>
+  <p>&#160;</p>
+  <p class="text">Date Induction Completed:&#160;<strong><%= qualification.details.periods.first.end_date.to_date.to_fs(:long_uk) %></strong></p>
```

Comment:
Induction periods are going away from the API very shortly; we should use the dates on the top-level induction property

Reply:
Should be fixed now